components/MeetingCard.vue

```diff
@@ -2,7 +2,8 @@
 	import type { Meeting } from '~/types'
 	const { display: displayDate, dayjs } = useDate()
 	const { deleteMeeting: deleteMeetingApi } = useApi()
-	const podStore = usePodStore()
+	const { getPod, isLeader } = usePodStore()
+	const pod = await getPod()
```

Comment:
You don't need to do `await getPod()` in the meeting card. We're calling `getPod` in `startApp` as soon as the page loads and storing the pod info inside of the PodStore.

You should be able to do 
```js
const {state: podState } = usePodStore()

<template>
  <p> {{ podState.id }} </p>
</template>
```